app/views/home/index.html.erb

```diff
@@ -1 +1,29 @@
-<h1>Hello World!</h1>
+<!DOCTYPE html>
+<%= stylesheet_link_tag 'index.css'%>
+<html>
+  <body>
+    <main class="card">
```

Comment:
Don't need any of this as this skeleton is provided by rails in `app/views/layouts/application.html.erb`.

Reply:
you're absolutely right, fixin it